src/styles/pages/get-involved/donate.module.scss

```diff
@@ -0,0 +1,51 @@
+.sectionContainer {
+  display: flex;
+  padding-top: 32px;
+  background-color: var(--color-white);
+  font-family: var(--font-inter);
+  padding: var(--edge-spacer);
+}
+
+.donateBox {
+  margin-top: 16px;
+  background-color: var(--color-beige);
+  padding-top: 16px;
+  padding-bottom: 16px;
+  padding-left: 32px;
+  padding-right: 32px;
+  border-radius: 5px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+  font-size: 1.25rem;
+}
+
+.donateHeading {
+  font-size: 2rem;
+  text-align: center;
+  color: var(--color-maroon);
+  padding-bottom: 16px;
+}
+
+.startSubtext1 {
+  padding-bottom: 16px;
+}
+
+.centeredSubtext {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.centeredTextBox {
+  text-align: left;
+}
+.centeredText1,
```

Comment:
If multiple divs or elements have the same styling, you can give the same classname to them. so if all the p tags for the address just have one classname like "centeredText" or "address", then you can target all of them with just one selector.